src/twisted/conch/newsfragments/9265.bugfix

```diff
@@ -0,0 +1 @@
+twisted.conch.client.default.py verifyHostKey now opens /dev/tty with no buffer to be compatible with Python 3.
```

Comment:
A stray `.py` suffix!  Also, please mention that the `conch` command line program works under Python 3 now as well.